src/Features/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -52,6 +52,11 @@ public static bool ShouldHideAdvancedMembers(this Document document)
             foreach (var linkedDocumentId in linkedDocumentIds)
             {
                 var linkedDocument = document.Project.Solution.GetDocument(linkedDocumentId);
+                if (!linkedDocument.SupportsSyntaxTree)
```

Comment:
Will send you a screen shot of the exception and call stack to see if there is a more targeted fix.